src/ProjectFileTools.MSBuild/MSBuildWorkspace.cs

```diff
@@ -0,0 +1,139 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.Build.Construction;
+using Microsoft.Build.Evaluation;
+using Microsoft.Language.Xml;
+
+namespace ProjectFileTools.MSBuild
+{
+    public class MSBuildWorkspace
```

Comment:
Remove MSBuild from name for MSBuildWorkspace and MSBuildWorkspaceManager (they are already part of the ProjectFileTools.MSBuild)